java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsContentAccess.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Android WebSettings content access
+ * @description Access to content providers in a WebView can enable JavaScript to access protected information.
+ * @kind problem
+ * @id java/android-websettings-content-access
+ * @problem.severity warning
+ * @security-severity 6.5
+ * @precision high
```

Comment:
Since this won't always be a security problem, let's get this down to `medium` so that the query becomes opt-in for users that want to see more results.